tasks/tool.py

```diff
@@ -132,9 +133,8 @@ def notify_macos(ctx, text):
     ctx.run(f"osascript -e '{CMD}' test-infra-definitions '{text}'")
 
 
-def notify_linux():
-    # TODO: Implement notification on linux. Would require linux computer (with desktop) to test
-    return
+def notify_linux(ctx, text):
+    ctx.run(f"notify-send 'test-infra-definitions' '{text}'")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is `notify-send` a system binary ? If not, can you add it as a dependency to the readme please ?

Reply:
Done in my last commit, although I think the `notify-send` binary comes preinstalled with most Linux distributions.